test/bigtest/helpers/index.js

```diff
@@ -0,0 +1,3 @@
+export { getCleanTestingRoot } from './getCleanTestingRoot';
+export { default as Harness } from './Harness';
+export { mountWithContext } from './mountWithContext';
```

Comment:
See above comment about copying-in code that can be imported from stripes-components.